### PR TITLE
Wildcard update

### PR DESCRIPTION
**Pull request by vyankyGH**
## Related issue #
Signed-off-by: Vyankatesh   <vyankateshkd@gmail.com>
closes https://github.com/kyverno/website/issues/494

## Checklist

- [x] I have read the [contributing guidelines](https://github.com/kyverno/website/blob/main/CONTRIBUTING.md).
- [x] I have inspected the website preview for accuracy.
- [x] I have signed off my issue.

### CODE REVIEW

File: content/en/docs/Writing policies/match-exclude.md [hunk 1]
```diff
@@ -41,6 +41,19 @@ These can be distinguished as:
...
 * `crd.antrea.io/v1alpha1/NetworkPolicy`
 
+Wildcards supported formats:
+
+* `Group/*/Kind`
+* `*/Kind`
+* `*`
+
+{{% alert title="Note" color="info" %}}
+* wildcard policy not allowed in background mode.
```
Comment (chipzoller) [inline content/en/docs/Writing policies/match-exclude.md:51]: "A policy using wildcards in `match` or `exclude` is not allowed in background mode."

File: content/en/docs/Writing policies/match-exclude.md [hunk 2]
```diff
@@ -41,6 +41,19 @@ These can be distinguished as:
...
+* wildcard policy does not support `Generate`, `Verify Images`, `ForEach` rule type.
```
Comment (chipzoller) [inline content/en/docs/Writing policies/match-exclude.md:52]: "A policy using wildcards does not `generate` or `verifyImages` rule types, and does not support `forEach` declarations."

File: content/en/docs/Writing policies/match-exclude.md [hunk 3]
```diff
@@ -41,6 +41,19 @@ These can be distinguished as:
...
+* In validate rule type, policy can only deal with Deny and the metadata field of the resource Pattern and AnyPattern.
```
Comment (chipzoller) [inline content/en/docs/Writing policies/match-exclude.md:53]: "For the `validate` rule type, a policy can only deal with `deny` statements and the `metadata` object in either `pattern` or `anyPattern`. blocks."

File: content/en/docs/Writing policies/match-exclude.md [hunk 4]
```diff
@@ -41,6 +41,19 @@ These can be distinguished as:
...
+* In Mutate rule type, policy can only deal metadata field.
```
Comment (chipzoller) [inline content/en/docs/Writing policies/match-exclude.md:54]: "For the `mutate` rule type, a policy can only deal with the `metadata` object."

**Review by chipzoller — CHANGES_REQUESTED** [review]
See in-line comments.

**Comment by vyankyGH** [comment]
> See in-line comments.

Thanks @chipzoller. I have addressed all comment.

File: content/en/docs/Writing policies/match-exclude.md [hunk 5]
```diff
@@ -41,6 +41,19 @@ These can be distinguished as:
...
 
+Wildcards supported formats:
+
+* `Group/*/Kind`
+* `*/Kind`
+* `*`
+
+{{% alert title="Note" color="info" %}}
+* A policy using wildcards in `match` or `exclude` is not allowed in background mode.
+* A policy using wildcards does not `generate` or `verifyImages` rule types, and does not support `forEach` declarations.
```
Comment (chipzoller) [inline content/en/docs/Writing policies/match-exclude.md:52]: Sorry, must have messed up:

"A policy using wildcards does not support `generate` or `verifyImages` rule types, and does not support `forEach` declarations."

File: content/en/docs/Writing policies/match-exclude.md [hunk 6]
```diff
@@ -41,6 +41,19 @@ These can be distinguished as:
...
+* For the `validate` rule type, a policy can only deal with `deny` statements and the `metadata` object in either  `pattern` or `anyPattern` blocks..
```
Comment (chipzoller) [inline content/en/docs/Writing policies/match-exclude.md:53]: Remove duplicate period at end of sentence.

**Review by chipzoller — CHANGES_REQUESTED** [review]
Comments

**Review by chipzoller — APPROVED** [review]

**Pull request merged**